lib_stusb_impl/usbd_impl.c

```diff
@@ -353,21 +353,33 @@ static uint8_t const HID_ReportDesc_fido[] = {
 
 #define ARRAY_U2LE(l) (l)&0xFF, (l)>>8
 
+#define CFG_HDR_LEN    (0x9)
+#define CFG_HIDGEN_LEN (0x9+0x9+0x7+0x7)
+#define CFG_IO_U2F_LEN (0x9+0x9+0x7+0x7)
+#define CFG_USB_CCID_LEN (0x9+0x36+0x7+0x7)
+#define CFG_WEBUSB_LEN (0x9+0x7+0x7)
+
 /* USB HID device Configuration Descriptor */
+#ifdef HAVE_USB_CLASS_CCID

```

Comment:
Similar question as above, is it necessary for all app using ccid, or is it an edge case of openpgp app and hence could be enabled under an additional app enabled defined?

Reply:
This is part of the CCID descriptor, not OpenPGP. I think we must keep it!